### ISSUE TITLE
Enrich the joining data with irregular time stamps

### ISSUE THREAD

**Issue by timhallinflux** [influxdata/docs-v2#1880]
_Describe the issue here._
Worth reviewing this additional guidance and ensuring that the documentation accurately reflects this.

If users want to join irregular data on time they must first normalize those time stamps.

There are two ways to do this depending on what the user actually wants:
1. Use truncateTimeColumn, this is useful when the user knows that there is noise in the time stamp itself but that they only expect a single point in a given time range.

2. Use aggregateWindow to first aggregate multiple points into a single result per time slot, the results of aggregateWindow will produce times on regular time intervals that can then be joined
Finally a side note but the experimental.join function will likely be faster in many cases

TL;DR use truncateTimeColumn when you expect a single point per time slot, use aggregateWindow when you expect multiple points per time slot

##### Relevant URLs
https://docs.influxdata.com/influxdb/v2.0/query-data/flux/manipulate-timestamps/#normalize-irregular-timestamps
